modules/aleph/src/Aleph/AlephDebt.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+use Drupal\aleph\Aleph\AlephDebtMaterial;
+
+/**
+ * Class AlephDebt.
+ *
+ * Implements a class with a patron's debt.
+ *
+ * @package Drupal\aleph\Aleph
+ */
+class AlephDebt {
+  protected $type;
+  protected $description;
+  protected $sum;
+  protected $date;
+  protected $debtMaterial;
+
+  /**
+   * Set the debt type.
+   *
+   * @param string $type
+   *    The debt type.
+   */
+  public function setType($type) {
+    $this->type = $type;
+  }
+
+  /**
+   * Get the debt type.
+   *
+   * @return string
+   *    The type of debt.
```

Comment:
A word or two about what a "type" is would be nice. Consider adding some examples of a type.